src/platform/user/profile/vap-svc/containers/CopyMailingAddress.jsx

```diff
@@ -58,27 +92,13 @@ export function mapStateToProps(state) {
     state,
     FIELD_NAMES.MAILING_ADDRESS,
   );
-  const hasEmptyMailingAddress = isEmptyAddress(mailingAddress);
-
-  const residentialAddress = selectEditedFormField(
-    state,
-    FIELD_NAMES.RESIDENTIAL_ADDRESS,
-  ).value;
-
-  const isChecked = () => {
-    if (hasEmptyMailingAddress) {
-      return false;
-    }
-    return isEqual(
-      pickBy(pick(mailingAddress, ADDRESS_PROPS), e => !!e),
-      pick(residentialAddress, ADDRESS_PROPS),
-    );
-  };
-
   return {
+    emptyMailingAddress: isEmptyAddress(mailingAddress),
```

Comment:
Why the rename of this prop?

Reply:
No specific reason, I  reorganized this file quite a bit and then made this change somewhere along the way, unaware of this change.